static/shared/js/typeahead.d.ts

```diff
@@ -0,0 +1,29 @@
+type Emoji = {
+    emoji_name: string;
+    emoji_code?: string;
```

Comment:
Under what circumstance would `emoji_code` be undefined? I think that all emoji types should have that field set unconditionally.

Reply:
`emoji_code` is only available for unicode emojis. We don't have that for realm emojis.